docs/sources/dashboards/share-dashboards-panels/_index.md

```diff
@@ -0,0 +1,372 @@
+1. Paste the HTML code into your website code.
+1. Click the **X** at the top-right corner to close the share drawer.
+
+Here's an example of what the HTML code might look like:
+
+```html
+<iframe
+  src="https://snapshots.raintank.io/dashboard-solo/snapshot/y7zwi2bZ7FcoTlB93WN7yWO4aMiz3pZb?from=1493369923321&to=1493377123321&panelId=4"
+  width="650"
+  height="300"
+  frameborder="0"
+></iframe>
+```
+
+The result is an interactive Grafana visualization embedded in an iframe.
+
+### Share a snapshot {#panel-snapshot}
+
+A panel snapshot shares an interactive panel publicly while removing sensitive data such as queries and panel links, leaving only visible metrics and series names. Anyone with the link can access the snapshot.
+
+You can publish snapshots to your local instance or to [snapshots.raintank.io](http://snapshots.raintank.io). The latter is a free service provided by Grafana Labs that enables you to publish dashboard snapshots to an external Grafana instance. Anyone with the link can view it. You can set an expiration time if you want the snapshot removed after a certain time period.
+
+{{< admonition type=note >}}
+The snapshots.raintank.io option is disabled by default in Grafana Cloud. You can update [your config file](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#external_enabled) to enable this functionality.
+{{< /admonition >}}
+
+To see the other snapshots shared from your organization, navigate to **Dashboards > Snapshots** in the main menu.
+
+To share your panel with anyone as a snapshot, follow these steps:
+
+1. Hover over any part of the panel you want to share to display the actions menu on the top-right corner.
+1. Click the menu and select **Share snapshot**.
+1. In the **Share snapshot** drawer that opens, enter a descriptive title for the snapshot in the **Snapshot name** field.
+1. Select one of the following expiration options for the snapshot:
+   - **1 Hour**
+   - **1 Day**
+   - **1 Week**
+   - **Never**
+1. Click **Publish snapshot** or **Publish to snapshots.raintank.io**.
+
+   Grafana generates the link of the snapshot. Note that you can't publish snapshots that include custom panels to snapshot.raintank.io.
+
+1. Click **Copy link**, and share it either within your organization or publicly on the web.
+1. Click the **X** at the top-right corner to close the share drawer.
+
+#### Delete a snapshot
+
+To delete existing snapshots, follow these steps:
+
+1. Navigate to **Dashboards > Snapshots** in the main menu.
+1. To confirm which snapshot you're about to delete, click **View** on the snapshot row.
+
+   The URLs for panel and dashboard snapshots from the same dashboard look similar and viewing them first can help you distinguish them.
+
+1. Click the red **x** next to the snapshot URL that you want to delete.
+
+The snapshot is immediately deleted. You may need to clear your browser cache or use a private or incognito browser to confirm this.
+
+### Share a library panel
```

Comment:
This isn't in the Share options, so should this even be on this page anymore? I think we either take it out entirely or at least remove the text and link folks to this page:
https://grafana.com/docs/grafana/latest/dashboards/build-dashboards/manage-library-panels/

Reply:
We can delete it since it is not part of the Share options anymore